crates/key-server/src/server.rs

```diff
@@ -621,6 +621,11 @@ async fn add_response_headers(mut response: Response) -> Response {
 #[tokio::main]
 async fn main() -> Result<()> {
     let master_key = env::var("MASTER_KEY").expect("MASTER_KEY must be set");
+    let bytes = if Base64::decode(&master_key).is_ok() {
+        Base64::decode(&master_key).expect("MASTER_KEY should be base64 encoded")
```

Comment:
why should we keep supporting b64?
and if we do, maybe let's make hex the default?

Reply:
because our own ci that already stores the base64 and have running instances. this is just to make the doc more consistent since genkey is changed.

Reply:
we should (later) change the ci to use hex and get rid of the old option